app/pages/home-for-user/news-pullout.styl

```diff
@@ -0,0 +1,46 @@
+.home-page-news-pullout
+
+  &__main
+    height: 100%
+    overflow: hidden
+    padding: 1em 2em
+    width: 100%
+
+    h2, h4
+      color: #0f7481
+      margin-bottom: 0.8em
+
+    a
+      color: inherit
+      text-decoration: none
+
+  &__container
+    height: 100%
+    overflow-y: scroll
+    width: 100%
+
+  &__section
+    display: flex
+    flex-direction: column
+    margin-bottom: 1.5em
+    max-width: 220px
+
+    &--timestamp
```

Comment:
The `--` in BEM is intended to indicate modifiers, so something like `.button--red` or `.button--blue`. In this case, these are elements of the section block. How I would approach this is to have section be its own block and timestamp, content, and title be its elements: `news-section__timestamp`. This is separate from the parent block, so the resulting div would have these classes: `home-page-news-pullout news-section__timestamp`